requirements.txt

```diff
@@ -1,4 +1,9 @@
-msgpack == 0.6
+# If you change the requirements here, don't forget to make the
+# corresponding change in the conda-forge rcpq-feedstock here:
+#
+# https://github.com/conda-forge/rpcq-feedstock/blob/master/recipe/meta.yaml
+
+msgpack >=0.6,<1.0
```

Comment:
Can we leave a note justifying this restriction with a link to the gh issue + reminder to maybe fix it one day.

Reply:
done